### DISCUSSION TITLE
How to upload an avatar to my account?

### DISCUSSION THREAD

**Discussion by nepomnyi** [c-ding-math/functor-network#18] [Q&A] [answered]
Hello, dear Functor Network community.
I am wondering how to upload an avatar picture on my account. When I go to the "About the author" tab and press "Edit", I can see the code for it. But I don't know how to use it. I am wondering if somebody can give me instructions, please.
Thank you in advance.
Ivan

**Comment by c-ding-math** [comment]
> Hello, dear Functor Network community. I am wondering how to upload an avatar picture on my account. When I go to the "About the author" tab and press "Edit", I can see the code for it. But I don't know how to use it. I am wondering if somebody can give me instructions, please. Thank you in advance. Ivan

Just replace the link in the code with the link of your avatar.

**Comment by nepomnyi** [comment]
That's exactly the part that I don't understand. What does "the link to my avatar" mean? My avatar is stored on my computer. I can't create a link to it. I can upload it from my computer, but there is no option for that.
I guess - to narrow down my question - I am wondering how to create a link to my avatar.
Thanks.
Ivan

**Reply by c-ding-math** [reply, marked as answer]
You can upload your avatar picture via the file library (click the "Files" menu in the navbar),  or use your existing avatar from any other place on the web.
